server/webdriver/screenshot/runner.py

```diff
@@ -25,23 +27,27 @@
 SCREENSHOTS_FOLDER = 'screenshots'
 
 
-def run(driver, url):
+def run(driver, base_url, page_config_dir):
   """Take screenshot and save to desired folders"""
   for f in os.listdir(SCREENSHOTS_FOLDER):
     if f != '.gitkeep':
       os.remove(os.path.join(SCREENSHOTS_FOLDER, f))
   curr_dir = os.path.dirname(os.path.abspath(__file__))
-  with open(os.path.join(curr_dir, 'page.json')) as f:
+  with open(os.path.join(curr_dir, page_config_dir, 'page.json')) as f:
     config = json.load(f)
     for page in config:
       # Set the window size. Testing different sizes.
       driver.set_window_size(width=WIDTH,
                              height=page['height'],
                              windowHandle='current')
-      driver.get(url + page['url'])
-      name = urllib.parse.quote_plus(page['url'].removeprefix('/'))
-      WebDriverWait(driver, shared.TIMEOUT).until(shared.charts_rendered)
+      driver.get(base_url + page['url'])
+      if page['async']:
```

Comment:
Include a comment in the code about what 'async' means? Because looking at the code, it's not very clear & I had to refer to the PR description

Reply:
Done